css/style.css

```diff
@@ -38,15 +37,186 @@ body {
 
 /* Center the container */
 
-.container {
+/* First Section */
+
+#wholepage {
+  width: 1435px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+
+.firstbox {
   width: 100%;
-  margin: auto;
-  height: 500px;
+  height: 600px;
+  background: url(../images/header_bg.jpg) no-repeat center top rgb(134, 141, 157);
+  background-size: cover;
+  border: 1px solid;
+}
+
+.firstp {
+  font-size: 24px;
+  font-weight: 800;
+  color: white;
+  margin-left: 300px;
+  padding-top: 30px;
+}
+
+.secondp {
+  font-weight: 600;
+  font-size: 24px;
+  color: white;
+  padding-top: 40px;
+}
+
+.firstbutton {
```

Comment:
Here, maybe you could call it just `.button` or `.primary-button`. The thing I'm suggesting you avoid is naming them based on their location. Its possible you'd want to reuse these styles to style another button later on in the page. but if that button is no longer the _first_ then then name doesn't make much sense.